core/src/main/java/org/apache/calcite/sql/type/ReturnTypes.java

```diff
@@ -396,6 +396,13 @@ public static SqlCall stripSeparator(SqlCall call) {
   public static final SqlReturnTypeInference DOUBLE_NULLABLE =
       DOUBLE.andThen(SqlTypeTransforms.TO_NULLABLE);
 
+  /**
+   * Type-inference strategy whereby the result type of a call is a nullable
+   * Double.
```

Comment:
Going forward if someone writes some code for you it's good practice to add them as a co-author. Don't worry about it for me but other people may want their work attributed to them.